controller/payload/payload_scheduler.py

```diff
@@ -183,6 +185,7 @@ def prepare_input_data_for_payload(
             training_record_ids = get_exclusion_record_ids(information_source_id)
             input_data = json.dumps(
                 {
+                    "information_source_id": str(information_source_item.id),
```

Comment:
why not `information_source_id` from create_payload? though i don't know if it's already a string however the typing suggest yes :D

Reply:
I don't know :D changed it